bundles/org.openhab.binding.meross/src/main/resources/OH-INF/thing/light.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="meross"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="light">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+
+		<label>Meross Light</label>
+		<description>Meross Light</description>
+
+		<channels>
+			<channel id="togglex" typeId="togglex"/>
```

Comment:
You could use the system state channel; `system.power` for reference: https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types
You only need to set the label and description

Reply:
I would leave it as is it, IMO such a refactor is not worth the effort, I would have to change tags and references too...